install-devstack-xen.sh

```diff
@@ -17,6 +18,10 @@ positional arguments:
                     This key will be copied over to the xenserver host, and will
                     be used for migration/resize tasks if multiple XenServers
                     used.  If '-' is passed, assume the key is provided by an agent
+ NODE_TYPE          OpenStack node type [controller, compute]
```

Comment:
controller includes compute, so instead of "controller" should be "all" or "complete"?